libmamba/src/core/activation.cpp

```diff
@@ -1215,4 +1215,99 @@ namespace mamba
 
         return out.str();
     }
+
+
+    std::string NuActivator::shell_extension()
+    {
+        return ".nu";
+    }
+
+    std::string NuActivator::shell()
+    {
+        return "nu";
+    }
+
+    std::string NuActivator::hook_preamble()
+    {
+        return "";
+    }
+
+    std::string NuActivator::hook_postamble()
+    {
+        return "";
+    }
+
+    // TODO: check this
+    fs::u8path NuActivator::hook_source_path()
+    {
+        return Context::instance().prefix_params.root_prefix / "etc" / "profile.d" / "micromamba.nu";
+    }
+
+    std::pair<std::string, std::string>
+    NuActivator::update_prompt(const std::string& /*conda_prompt_modifier*/)
+    {
+        return { "", "" };
+    }
+
+    // TODO: check this
+    std::string NuActivator::script(const EnvironmentTransform& env_transform)
+    {
+        std::stringstream out;
+        auto convert_to_nulist = [](std::string command)
+        {
+            std::string new_command = command;
+            std::size_t idx = 0;
+            bool add = false;
+            while ((idx = new_command.find(":", idx)) != std::string::npos)
+            {
+                new_command.replace(idx, 1, " ");
+                add = true;
+                idx++;
+            }
+            if (add)
+            {
+                new_command = "[" + new_command + "]";
+            }
+            return new_command;
+        };
+
+        if (!env_transform.export_path.empty())
+        {
+            out << "$env.PATH = [" << env_transform.export_path << "]\n";
```

Comment:
I think it would be preferable to use `fmt::format` for this kind of formatting but it looks like the rest of this file also doesn't do that